<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGui::FileBrowserGeneralOptionPage</class>
 <widget class="QWidget" name="QtGui::FileBrowserGeneralOptionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox { font-weight: bold };</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="miscGroupBox">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="hideBackupFilesCheckBox">
        <property name="text">
         <string>Hide backup/temporary files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="readOnlyCheckBox">
        <property name="text">
         <string>Read only</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
